src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -42,7 +46,7 @@ public partial class BrotliStream : Stream
         }
         public BrotliStream(Stream baseStream, CompressionMode mode, bool leaveOpen, int BuffSize)
         {
-            if (baseStream == null) throw new ArgumentNullException("baseStream");
+            if (baseStream == null) throw new ArgumentNullException(BrotliEx.baseStream);
```

Comment:
These types of strings don't need to be in resources... because they need to match the parameter names. They should stay in code.